packages/analytics/addon/components/ca-report-preview.hbs

```diff
@@ -30,7 +30,10 @@
         {{#each this.data.value.fields as |row|}}
           <tr>
             {{#each row as |entry|}}
-              <td data-t={{this.getXLSXType entry.value}}>
+              <td
+                data-t={{this.getXLSXType entry.value}}
+                data-v={{this.cleanValue entry.value}}
```

Comment:
Where are those attributes consumed? Couldn't find any docs. :man_shrugging:

Reply:
Used in the excel export of the analytics table. Definition is here https://docs.sheetjs.com/docs/csf/cell

Reply:
Thanks! After skimming over the dox I've found this one to be even more appropriate: https://docs.sheetjs.com/docs/api/utilities/html#implementation-details